sources/api/migration/migrator/tests/data/fake-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
We call this `snakeoil.pem` [in tough](https://github.com/awslabs/tough/blob/develop/tough/tests/data/snakeoil.pem), might be nice to do the same here so it's easier to identify keys made for test purposes.

Reply:
Addressed in push to [00ce955](https://github.com/bottlerocket-os/bottlerocket/compare/f5a28e55ec6f9ebfde8b8e720b0aee0689dee61b..00ce9553e6dfcb1dd09b35361e478cd29502f2bb)